src/main/java/mil/dds/anet/beans/Person.java

```diff
@@ -307,6 +311,19 @@ public void setCode(String code) {
     this.code = code;
   }
 
+  @JsonIgnore
+  public List<Map<String, String>> getUserActivities() {
```

Comment:
LGTM warning: "getUserActivities exposes the internal representation stored in field userActivities. The value may be modified", please fix.